libs/common/src/helpers/round-number.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Rounds a number to a specified number of digits.
+ * @param value value to round
+ * @param digits number of digits to round to, defaults to 2 if not provided
+ * @returns rounded number
+ */
+export const roundNumber = (value: number, digits?: number) => {
```

Comment:
@zainUKEF why you don't just say. if digits is 0 then the d will be 2
```suggestion
export const roundNumber = (value: number, digits: number = 2) => {
```

Reply:
done